src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --link-on: #00acdc;
+  --link-off: black;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  height: 60px;
```

Comment:
[Requirements:](https://github.com/mate-academy/layout_moyo-header/blob/master/readme.md#requirements) "the height should be set for nav links (**not the header**), take it from the design"